.travis.yml

```diff
@@ -2,15 +2,12 @@ sudo: false
 
 language: go
 
-go:
-        - 1.4
-        - 1.5
-        - 1.6
+branches:
+        only:
+                - merkle-tree
```

Comment:
Travis has a `Build only if .travis.yml is present` setting, but I see you've already pushed one to master. No bother.